src/controllers/subject.js

```
@@ -18,12 +18,16 @@ const getSubjects = async (req, res) => {
 
 const createSubject = async (req, res) => {
   const subjectData = req.body;
+  
+  const userRole = req.user?.role;
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)